test/integration/concrete/lockup/withdraw/withdraw.tree

```diff
@@ -8,90 +8,93 @@ withdraw.t.sol
-                      │                ├── it should call the sender hook
-                      │                ├── it should emit a {MetadataUpdate} event
-                      │                └── it should emit a {WithdrawFromLockupStream} event
-                      └── when the caller is an approved third party
+                └── when the withdraw amount does not overdraw
+                   ├── given the withdrawal address is not the stream recipient
+                   │  ├── when the caller is the sender
+                   │  │  └── it should revert
+                   │  ├── when the caller is unknown
+                   │  │  └── it should revert
+                   │  ├── when the caller is a former recipient
+                   │  │  └── it should revert
+                   │  ├── when the caller is the recipient
+                   │  │  ├── given the sender is not a contract
+                   │  │  │  ├── it should make the withdrawal
+                   │  │  │  └── it should update the withdrawn amount
+                   │  │  └── given the sender is a contract
+                   │  │     ├── given the sender does not implement the hook
+                   │  │     │  ├── it should make the withdrawal
+                   │  │     │  ├── it should update the withdrawn amount
+                   │  │     │  ├── it should call the sender hook
+                   │  │     │  └── it should ignore the revert
+                   │  │     └── given the sender implements the hook
+                   │  │        ├── when the sender reverts
+                   │  │        │  ├── it should make the withdrawal
+                   │  │        │  ├── it should update the withdrawn amount
+                   │  │        │  ├── it should call the sender hook
+                   │  │        │  └── it should ignore the revert
+                   │  │        └── when the sender does not revert
+                   │  │           ├── when there is reentrancy
+                   │  │           │  ├── it should make multiple withdrawals
+                   │  │           │  ├── it should update the withdrawn amounts
+                   │  │           │  └── it should call the sender hooks
+                   │  │           └── when there is no reentrancy
+                   │  │              ├── it should make the withdrawal
+                   │  │              ├── it should update the withdrawn amount
+                   │  │              ├── it should call the sender hook
+                   │  │              ├── it should emit a {MetadataUpdate} event
+                   │  │              └── it should emit a {WithdrawFromLockupStream} event
+                   │  └── when the caller is an approved third party
+                   │     ├── it should make the withdrawal
+                   │     └── it should update the withdrawn amount
+                   └── given the withdrawal address is the stream recipient
```

Comment:
This has be "when", not "given", because `to` is a user-provided parameter.

Reply:
Right. I made an error there.